aws_lambda_powertools/utilities/data_classes/common.py

```diff
@@ -19,6 +19,11 @@ def __eq__(self, other: Any) -> bool:
     def get(self, key: str) -> Optional[Any]:
         return self._data.get(key)
 
+    @property
+    def data(self) -> Dict[str, Any]:
```

Comment:
Perhaps raw_data? Or raw_event?

Reason being is that data is such a common name and we might have collision